sentry/src/main/java/io/sentry/ISpan.java

```diff
@@ -0,0 +1,21 @@
+package io.sentry;
+
+/** Represents performance monitoring Span. */
+public interface ISpan {
+  /**
+   * Starts a child Span.
+   *
+   * @return a new transaction span
+   */
+  Span startChild();
+
+  /**
+   * Returns a string that could be sent as a sentry-trace header.
+   *
+   * @return string containing sentry-trace header.
+   */
+  String toTraceparent();
```

Comment:
I propose we rename this.

This is what traceparent is: https://www.w3.org/TR/trace-context/#traceparent-header

Naming this here is very confusing, stops us from being able to actually return `traceparent` in the future with this function name.

What about naming it as it is: `toSentryTrace`?